### DISCUSSION TITLE
ray()->showSlowQueries()

### DISCUSSION THREAD

**Discussion by dillingham** [spatie/laravel-ray#230] [Ideas]
```php
ray()->showSlowQueries($time = '0.5')
```

**Comment by freekmurze** [comment]
Would be cool to have this indeed! I'd accept a PR that adds this. If none is submitted, I'll take care of it myself soon!

**Comment by patinthehat** [comment]
@dillingham @freekmurze I've opened a PR for this feature, #232.

**Reply by dillingham** [reply]
@patinthehat thanks for PRing that!

**Reply by patinthehat** [reply]
@dillingham No problem - happy to help!
